specification/repository.md

```diff
@@ -136,20 +136,26 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
 
 ## Data Collector
 
-- MUST document all supported configuration parameters
+- MUST document all configuration parameters that are relevant to Splunk Observability Cloud
+- MUST document all configuration parameters whose default values deviate significantly from upstream
```

Comment:
What does "significantly" mean here?

Reply:
I would also add that sometimes we can also support a different set of values than the upstream 😉

Reply:
@pellared How would you phrase that?

@MrAlias I hear you on the adverb. Removing it.

Reply:
E.g.

```
MUST document all configuration parameters whose default or accepted values deviate from upstream
```

Reply:
I like that wording! Replacing.